README.md

```diff
@@ -79,6 +91,11 @@ Reference:
   - `generator`, _(optional)_ use a generator for the value, currently only support `secret`
 - `options`: _(optional)_ Options when deploying the service
   - `allow-unauthenticated`: _(optional, default: `true`)_ allow unauthenticated requests
+- `hooks`: _(optional)_ Run commands in separate bash shells with the environment variables configured for the application and environment variables for `PROJECT_ID`, `K_SERVICE`, and `REGION`. Command outputs are shown as they are executed.
```

Comment:
just a final thought here:

- I think if we be more specific, like GOOGLE_CLOUD_PROJECT and GOOGLE_CLOUD_REGION, that would be better. (as a matter of fact many of our client libs support GOOGLE_CLOUD_PROJECT ootb)

- please do 80-char line wraps (use vscode + markdownlint + rewrap)